workshop/Lab3/README.md

```diff
@@ -68,35 +70,37 @@ spec:
           containerPort: 3000
 ```
 
-The above configuration file create a deployment object named 'guestbook'
+The above configuration file creates a deployment object named `guestbook`
 with a pod containing a single container running the image
-`ibmcom/guestbook:v1`.  Also the configuration specifies replicas set to 3
-and Kubernetes tries to make sure that at least three active pods are running at
+`ibmcom/guestbook:v1`. The configuration also specifies a replica number
```

Comment:
like above "replica number" could be misleading... "replica count" would be ok though if you don't to go back to the existing words.

Reply:
Okay. Done.